src/main/java/cz/cvut/kbss/study/rest/PatientRecordController.java

```diff
@@ -62,15 +57,48 @@ public List<PatientRecordDto> getRecords(
 
     @PreAuthorize(
             "hasRole('" + SecurityConstants.ROLE_ADMIN + "') or @securityUtils.isMemberOfInstitution(#institutionKey)")
-    @GetMapping(value = "/export", produces = MediaType.APPLICATION_JSON_VALUE)
-    public List<PatientRecord> exportRecords(
+    @GetMapping(value = "/export", produces = {MediaType.APPLICATION_JSON_VALUE, Constants.MEDIA_TYPE_EXCEL})
+    public ResponseEntity<?> exportRecords(
```

Comment:
We just need to make sure that we did not change implementation of JSON export.

Reply:
@LaChope please test it when implementing UI

Reply:
I exported twice, before and after this PR, output JSON file is similar